stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/internal/MessageListItemViewTypeMapper.kt

```diff
@@ -52,6 +54,7 @@ internal object MessageListItemViewTypeMapper {
             message.deletedAt != null -> MESSAGE_DELETED
             message.isGiphyEphemeral() -> GIPHY
             containsGiphy -> GIPHY_ATTACHMENT
+            message.attachments.all { it.isImage() || it.hasLink() } -> IMAGE_ATTACHMENT
```

Comment:
why do we return true if the any of the items `hasLink()`?

Reply:
I changed the condition a bit